pkg/cli/clusteroperator/restartkubelet/restart-pod-template.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  generateName: restart-kubelet-pod-
+spec:
+  restartPolicy: Never
+  containers:
+    - command:
+        - sleep
```

Comment:
Presumably something will replace this?  What?

Reply:
> Presumably something will replace this? What?

Whatever Ryan ends up developing.  :)